tests/integration/helpers/cluster.py

```diff
@@ -154,14 +159,15 @@ def __init__(self, base_path, name=None, base_configs_dir=None, server_bin_path=
 
         self.docker_client = None
         self.is_up = False
+        print "CLUSTER INIT base_config_dir:{} config_dir:{}".format(self.base_config_dir, self.config_dir)
 
     def get_client_cmd(self):
         cmd = self.client_bin_path
         if p.basename(cmd) == 'clickhouse':
             cmd += " client"
         return cmd
 
-    def add_instance(self, name, config_dir=None, main_configs=None, user_configs=None, macros=None,
+    def add_instance(self, name, base_config_dir=None, config_dir=None, main_configs=None, user_configs=None, dictionaries = None, macros=None,
```

Comment:
The goal of this your PR is to stop using the `config_dir` parameter, isn't it? So it seems that parameter could be removed.

Reply:
Exactly! Done